.storybook/manager.js

```diff
@@ -0,0 +1,12 @@
+import {addons, types} from '@storybook/addons'
+import {Tool, TOOL_ID, ADDON_ID} from './src/accessibility-tool'
+
+addons.register(ADDON_ID, () => {
+  // Register the tool
+  addons.add(TOOL_ID, {
+    type: types.TOOL,
+    title: 'My addon',
```

Comment:
Nitpick, but it would be nice to name this and give it a unique id based on that name